.vib/clickhouse-operator/runtime-parameters.yaml

```diff
@@ -0,0 +1,231 @@
+# Adding an extra initial delay to avoid race conditions with the
+# deployment and Goss stages (so the operator has time to deploy
+# ClickHouse and ClickHouse Keeper installations)
+readinessProbe:
+  initialDelaySeconds: 45
+auth:
+  username: test_user
+  password: bitnami1234
+extraDeploy:
+# ref: https://github.com/Altinity/clickhouse-operator/tree/master/docs/chi-examples
+- |
+  apiVersion: clickhouse.altinity.com/v1
+  kind: ClickHouseInstallationTemplate
+  metadata:
+    name: default-chi-template
+    namespace: {{ include "common.names.namespace" . | quote }}
+  spec:
+    templates:
+      podTemplates:
+        - name: default-clickhouse
+          distribution: Unspecified
+          metadata:
+            labels: {{- include "common.labels.matchLabels" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 12 }}
+              app.kubernetes.io/part-of: clickhouse-operator
+              app.kubernetes.io/component: clickhouse
+          spec:
+            securityContext:
+              fsGroup: 1001
+              fsGroupChangePolicy: Always
+              supplementalGroups: []
+              sysctls: []
+            containers:
+              - name: clickhouse
```

Comment:
If you don't set the image pull secrets this will not work in our internal pipelines

Reply:
done at https://github.com/bitnami/charts/pull/32816/commits/f07d8c5a0c14c4e10078b83f623887f6736a00ff